src/networks/standalone_sgan/sgan.py

```diff
@@ -0,0 +1,275 @@
+import argparse
+import os
+import numpy as np
+import torchvision.transforms as transforms
+from torchvision.utils import save_image
+from torch.utils.data import DataLoader
+from torchvision import datasets
+from torch.autograd import Variable
+import torch.nn as nn
+import torch
+import logging
+import time
+
+from networks.standalone_sgan.score_factory import ScoreCalculatorFactory
+
+directory_or_file_name = int(time.time() * 1000)
```

Comment:
Use more method so it can be used as a module. E.g. specify `if __name__ == '__main__':` as entry point